src/coreclr/jit/morph.cpp

```diff
@@ -17681,6 +17681,9 @@ void Compiler::fgExpandQmarkForCastInstOf(BasicBlock* block, Statement* stmt)
 
     assert(qmark->gtFlags & GTF_QMARK_CAST_INSTOF);
 
+    float currBbWeight = block->bbWeight;
```

Comment:
float => weight_t

Reply:
Also please use `const` as it makes it easier to see this isn't going to be messed with below.